### ISSUE TITLE
Create an initial FAQ Section

### ISSUE THREAD

**Issue by joewxboy** [open-horizon/open-horizon.github.io#18]
Create the initial central FAQ page (including navigation to that page) which should consist of a short description, categories, and short questions in each section.  This may require creation of a new FAQ page type.  Investigate using front matter to specify FAQ page type, categorization, and keywords with an eye towards eventually using the keywords for basic searching in a future phase.  For now, let's assume that when we translate a page in a future phase, we use the pagename.language.extension naming scheme, ex: `index.md` would be the default US English language version, `index.fr.md` would be a French version, and `index.zhcn.md` would be a Simplified Chinese version.  For now, I'm assuming that we'll start with one page per category, unless you determine that the pages would be too large.

As we build the pages up, let's document the decision points made and why they were made, and ensure that others who come along later to maintain can continue using the same practices and structures that we develop here.

Also left to a future phase (out of the scope of this issue) are using/specifying `defined variables`.

**Comment by joewxboy** [comment]
It looks like we have two options for creating FAQ entries: [posts](https://jekyllrb.com/docs/posts/) and [collections](https://jekyllrb.com/docs/collections/).  It appears that collections require external configuration versus posts allowing internal classification.

Within posts, you can organize them by category or tag.  Categories allow a post to belong to more than one at a time, but they are stacked, which is less flexible.  They also require the markdown doc to live in a folder named after the category, which precludes a post from belonging to more than one set of stacked categories.  Therefore, it appears that tags offer the most flexible arrangement (assuming that we want FAQ entries living in more than one classification area.

In summary, I _think_ we want the central FAQ page to be a list of tags and maybe the first (or recent) N (five?) FAQs in that tag to be displayed.  The FAQ category pages would then be one page per tag, which list out the FAQs with that tag (in what order?). This would also allow us to have one page per FAQ, if we wanted to or needed it.  Posts also allow us to have a shorter and longer answer using the excerpt separator, which would allow us to have a short answer on the central FAQ page and the FAQ category page (if we wanted this), and a longer answer on a dedicated FAQ page.

Thoughts, comments?

**Comment by xiuleiyy** [comment]
@joewxboy wow. Although posts are used to post blogs/papers in most instances. Your idea to use it for FAQ category/filter by tag is very interesting and I like it. Let's do this.
